src/data/real/nnreal.lean

```diff
@@ -635,4 +635,17 @@ mt pow_eq_zero h
 
 end pow
 
+section abs
+
+@[simp] lemma abs_eq (x : ℝ≥0) : abs (x : ℝ) = x :=
+abs_of_nonneg x.property
+
+/-- The absolute value on `ℝ` as a map to `ℝ≥0`. -/
+protected def abs : ℝ → ℝ≥0 := λ x, ⟨abs x, abs_nonneg x⟩
```

Comment:
I find it odd that `nnreal.abs` is a function `real -> nnreal`. Should this be called something like `real.nonneg_abs` instead?

Reply:
`real.nnabs`?

Reply:
I switched to Sébastien's `real.nnabs`.